config/overlays/mutation_webhook/kustomization.yaml

```diff
@@ -0,0 +1,20 @@
+# TODO: this is a temporary kustomization for the mutation webhook
+# It is kept separate until the mutation feature is stable enough,
+# when the mutation webhook should be moved to config/webhooks/manifests.yaml  
+
+namespace: gatekeeper-system
+
+bases:
+  - ../dev
+
+resources:
+  - webhook.yaml
+
+patchesJson6902:
+- target:
+    group: rbac.authorization.k8s.io
+    version: v1
+    kind: ClusterRole
+    name: manager-role
+  path: webhook_permissions_patch.yaml
+  
```

Comment:
nit: remove this extra new line

Reply:
Done

Reply:
I think there may be some dangling whitespace on the last line, that is why we are getting the red error notification here

Reply:
Should be better now